### PR TITLE
CODE RUB: Rename Brokers/Logging to Brokers/Loggings

### LINKED ISSUE #35: CODE RUB: Rename Brokers/Logging to Brokers/Loggings

### PR DESCRIPTION
**Pull request by talatmorshedtanbir**
closes #35

### CODE REVIEW

**Comment by ShriHumrudha** [comment]
We need to change the folder name as well.

**Comment by talatmorshedtanbir** [comment]
> We need to change the folder name as well.

Already changed the folder name from Logging to Loggings

**Comment by ShriHumrudha** [comment]
Please fix the build errors.

**Comment by talatmorshedtanbir** [comment]
Fixed build errors

File: OtripleS.Portal.Web/Services/Students/StudentService.cs [hunk 1]
```diff
@@ -5,8 +5,8 @@
 
 using System.Collections.Generic;
 using System.Threading.Tasks;
+using OtripleS.Portal.Web.Brokers.Loggings;
```
Comment (ShriHumrudha) [inline OtripleS.Portal.Web/Services/Students/StudentService.cs:8]: Please run code cleanup. It will order the directives in alphabetical order.

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Portal.Web/Services/Students/StudentService.cs (hunk 1 above)
Comment (talatmorshedtanbir) [reply OtripleS.Portal.Web/Services/Students/StudentService.cs:8]: You mean OtripleS directives will be above System directives? In other PR I was told to keep System directives above all

Comment (talatmorshedtanbir) [reply OtripleS.Portal.Web/Services/Students/StudentService.cs]: Ran code cleanup to order the directives.

File: OtripleS.Portal.Web/Services/StudentViews/IStudentViewService.cs [hunk 2]
```diff
@@ -3,8 +3,8 @@
 // FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
 // ---------------------------------------------------------------
 
-using System.Threading.Tasks;
 using OtripleS.Portal.Web.Models.StudentViews;
+using System.Threading.Tasks;
```
Comment (ShriHumrudha) [inline OtripleS.Portal.Web/Services/StudentViews/IStudentViewService.cs:7]: You need to enable a setting to ensure 'System' directives are already on the top.

Reference: https://dailydotnettips.com/automatically-place-the-system-directives-first-when-sorting-usings-option-in-visual-studio/

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

File: OtripleS.Portal.Web/Services/StudentViews/IStudentViewService.cs (hunk 2 above)
Comment (talatmorshedtanbir) [reply OtripleS.Portal.Web/Services/StudentViews/IStudentViewService.cs]: Thanks for the suggestion. Fixed it

**Pull request merged**